src/pages/components/menu-buttons/usage.mdx

```diff
@@ -0,0 +1,621 @@
+triggered when the user interacts with it. The text label must accurately
+describe the button action or indicate that interacting with it displays a list
+of possible actions.
+
+### Further guidance
+
+For further content guidance, see Carbon's
+[content guidelines](/guidelines/content/overview).
+
+## Universal behaviors
+
+### States
+
+The default and interactive states of menu buttons should follow the button
+style guidelines.
+
+- Combo buttons are available only as primary buttons.
+- Menu buttons come in three different button types: primary, tertiary, and
+  ghost.
+- Overflow button is treated as the ghost button, displaying only an icon and
+  following its designated color and style.
+
+### Interactions
+
+#### Menu button
+
+Users can reveal the menu by:
+
+- Clicking anywhere on the field of a menu button
+- Or tabbing into the menu button and pressing `Return` or `Enter`
+
+<Row>
+<Column colLg={8}>
+
+![Menu button interactions](images/usage-menu-keyboard-menu-button.png)
+
+<Caption>Mouse and keyboard interactions for menu button</Caption>
+</Column>
+</Row>
+
+#### Combo button
+
+Users can reveal the menu by:
+
+- Clicking anywhere on the field of the primary button or the icon button. If
+  clicking on the primary button, this will apply to certain elements, depending
+  on user interaction.
+- Or tabbing into the combo button which will focus on the primary button first
+  then the icon button. Then, pressing the `Return` or `Enter` to trigger the
+  action of the primary button or menu under icon button.
+
+<Row>
+<Column colLg={8}>
+
+![Combo button mouse interaction](images/usage-menu-mouse-combo-button.png)
+
+<Caption>Mouse interactions for combo button</Caption>
+</Column>
+</Row>
+
+<Row>
+<Column colLg={8}>
+
+![Combo button keyboard interaction](images/usage-menu-keyboard-combo-button.png)
+
+<Caption>Keyboard interactions for combo button</Caption>
+</Column>
+</Row>
+
+#### Overflow menu
+
+Users can reveal the overflow menu by:
+
+- Clicking anywhere on the field of the icon button
+- Or tabbing into the icon button and pressing `Return` or `Enter`
+
+<Row>
+<Column colLg={8}>
+
+![Overflow menu mouse and keyboard interactions](images/usage-menu-mouse-keyboard-overflow-menu.png)
+
+<Caption>Keyboard interactions for overflow menu</Caption>
+</Column>
+</Row>
+
+## Menu button
+
+Menu buttons group actions that share the same level of importance. The icon
+used within the menu button should always be a chevron in order to visually
+convey to the user that there are options to select from. There are three main
+use cases where using a menu button is recommended:
+
+- Grouping objects with the same common action
+- Actions related to a common title
+- Distinct actions applied to the same object or page
+
+Menu button and combo button come in three different button types: primary,
+tertiary, and ghost.
+
+<Row>
+<Column colLg={8}>
+
+![An example of menu button](images/usage-menu-button-1.png)
```

Comment:
The menu should be using `$layer-02` background here since its opening on top of $layer-01.